templates/extra/privacy.html

```diff
@@ -1,50 +1,50 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1 class="privacy-title">Privacy</h1>
+    <h2 class="privacy-title">Privacy</h2>
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
+    <div class="extra-padding-wide">
         <h5>PRIVACY POLICY</h5>
         <h5>LAST UPDATED: JULY 11, 2016</h5>
         <h5>PerDiem, LLC ("us", "we", or "our") operates the http://www.investperdiem.com website (the "Service").</h5>
-        <p>This page informs you of our policies regarding the collection, use and disclosure of Personal Information when you use our Service.</p>
-        <p>We will not use or share your information with anyone except as described in this Privacy Policy.
-        <p>We use your Personal Information for providing and improving the Service. By using the Service, you agree to the collection and use of information in accordance with this policy. Unless otherwise defined in this Privacy Policy, terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, accessible at http://www.perdiem.rocks</p>
+        <p class="left-align">This page informs you of our policies regarding the collection, use and disclosure of Personal Information when you use our Service.</p>
```

Comment:
Do not apply `class="left-align"` to every `<p>` tag. Do what you did in `faq.html`.